### PR TITLE
Fix alert styles

### PR DESCRIPTION
**Pull request by hbillings**
Some `usa-alert` classes need to be extended to Django default classes; others (like icon placement) need adjusting.

### CODE REVIEW

**Comment by jseppi** [comment]
I'm still seeing weird icon placement on `messagelist` "alerts" in the Django admin:

screen shot 2018-04-03 at 9 27 36 am

Maybe we could just remove the icons in `messagelist`s?

**Comment by jseppi** [comment]
wait, that's not even the one this PR was addressing. Do you have an example of how to find/generate a `message-info`?

File: frontend/source/sass/components/_alerts.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@ $color-error-bg: #fef5f5;
   margin: 1rem 0;
   padding: 1rem 6rem 1rem 2rem;
   border-radius: $border-radius;
+  background-image: none;
```
[the remark this answers is not in the document]
Comment (hbillings) [reply frontend/source/sass/components/_alerts.scss]: It removes them from all alerts except the error alert you get on price list upload. We'd removed them all before, but WDS changed the way they displayed the background image. I think whether or not we want it is an @ericronne question; will request his review!

[the remark this answers is not in the document]
Comment (ericronne) [reply frontend/source/sass/components/_alerts.scss]: Alert icons should stay! Long live alert icons! Donkeys are another story.

**Comment by hbillings** [comment]
@ericronne Okay, if we're keeping [alert icons from WDS](https://designsystem.digital.gov/components/alerts/), I need guidance on when to use what. For instance, right now we're using the "warning" icon (but in red) on the error message we display on Step 3 of the data upload if there are errors.
screen shot 2018-04-03 at 10 48 16 am

**Comment by hbillings** [comment]
Some other alert instances:
screen shot 2018-04-03 at 10 52 35 am
screen shot 2018-04-03 at 10 51 22 am
screen shot 2018-04-03 at 10 50 49 am
screen shot 2018-04-03 at 10 50 05 am
screen shot 2018-04-03 at 10 48 16 am

**Comment by ericronne** [comment]
The big red type in a red box is different than my original design, but i hadn't addressed alert design fully. I made this little reference, but frankly i don't recall how these were to be implemented. Tags? How about a refresher hangout?

image

**Comment by hbillings** [comment]
@ericronne Oh, those are tags that appear next to price list uploads for contracting officers. They look like this (and still have their logos):
screen shot 2018-04-03 at 11 25 10 am

I'm not sure you ever styled the alerts?

**Comment by hbillings** [comment]
(Also, you're right, the step 3 error alert shouldn't have a red background. I'll fix that.)

**Comment by ericronne** [comment]
Can you hold off for a few? ...

**Comment by ericronne** [comment]
Would love to give you a full set of alert designs.

**Comment by hbillings** [comment]
Could we merge this fix so that alerts aren't broken? They're currently illegible. Then we can open a new PR for new alert styles.

**Comment by ericronne** [comment]
Since i can't easily preview the changes … i gather the screenshots above represent the changes? I.e. no icons, black text, tint fills, and darker-color outlines?

**Comment by hbillings** [comment]
Oh, sorry! No, those are the styles that existed before. The fixes in here mostly apply to stuff the WDS changed -- i.e., the styles were no longer consistently applied, and the icons that we had previously hidden reappeared to look like this:
screen shot 2018-04-03 at 11 44 47 am

**Comment by ericronne** [comment]
🙊 
have a shot of the fixed version?

**Comment by hbillings** [comment]
Yeah, those are the screenshots I pasted above. :)

**Comment by hbillings** [comment]
Fixed the step 3 so that it looks like this now:
screen shot 2018-04-03 at 11 49 09 am

Awesome, I think these need some solid work and I'm happy to put in the time on them!

**Comment by ericronne** [comment]
Does this close #1689?

**Comment by ericronne** [comment]
or rather, #1686??

**Comment by hbillings** [comment]
yep, good call!

**Pull request merged**